src/OpenTelemetry.Api/Context/Propagation/IPropagator.cs

```diff
@@ -20,8 +20,8 @@
 namespace OpenTelemetry.Context.Propagation
 {
     /// <summary>
-    /// Text format wire context propagator. Helps to extract and inject context from textual
-    /// representation (typically http headers or metadata collection).
+    /// Defines an interface for Propagator, which is used to read and write
```

Comment:
Minor: consider the convention from https://docs.microsoft.com/dotnet/api/system.collections.generic.icollection-1